primitives/tests/light_client/validator_set.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
```

Comment:
I guess this file will have some stuff related to the validator set in the future?

Reply:
Well, it contains the `Signature` and `Public` key of the validator set already, I don't expect there is going to be much more logic (we could encode set transitions here though if needed, but I find it simpler to put directly in the LC code for now).

Reply:
I only asked this because of the file name 😛